azurerm/resource_arm_loadbalancer.go

```diff
@@ -114,6 +119,7 @@ func resourceArmLoadBalancerCreate(d *schema.ResourceData, meta interface{}) err
 	name := d.Get("name").(string)
 	location := d.Get("location").(string)
 	resGroup := d.Get("resource_group_name").(string)
+	sku := convertSkuStringToLoadBalancerSku(d.Get("sku").(string))
```

Comment:
rather than parsing this out I think we can make this `network.LoadBalancerSkuName(d.Get("sku").(string))`?